src/rgw/rgw_crypt.cc

```diff
@@ -1090,6 +1092,78 @@ int rgw_s3_prepare_encrypt(struct req_state* s,
       }
     }
 
+    /* Checking bucket attributes if SSE is enabled. Currently only supporting SSE-S3 */
+    rgw::sal::Attrs buck_attrs(s->bucket_attrs);
+    auto aiter = buck_attrs.find(RGW_ATTR_BUCKET_ENCRYPTION);
+    if (aiter != buck_attrs.end()) {
+      ldpp_dout(s, 5) << "Found RGW_ATTR_BUCKET_ENCRYPTION attr bucket_name: "
```

Comment:
Found -> found
Usually the small case is followed in the codebase unless we put it as Error or Notice.